pkgs/applications/emulators/rpcs3/default.nix

```diff
@@ -32,8 +24,6 @@ stdenv.mkDerivation {
     sha256 = rpcs3Sha256;
```

Comment:
```suggestion
     hash = rpcs3Hash;
```

Reply:
Can regular `hash` detect that's a sha256 in that format?

Reply:
It cannot, I'll have to update `update.sh`
```
error: hash '14k7dla6m7238kvxddvvw2f17gbg2pw0rf7d8xlz6ph95i1dlcsg' does not include a type, nor is the type otherwise known from context
```

Reply:
Convert to SRI. 

Worst case you can prepend `sha256:` before the hash.

Reply:
Yup, I'm running it through `nix hash to-sri --type sha256`, just rerunning the update script to check it works.